src/port.js

```diff
@@ -0,0 +1,21 @@
+'use strict'
+
+class Port {
+    constructor({ name }) {
+        this.name = name;
+        this._ships = [];
+    }
+
+    addShip(ship) {
+        this._ships.push(ship);
+    }
+
+    removeShip(ship) {
+        const removedShip = this._ships.findIndex((el) => {
```

Comment:
Brackets arent needed here
```suggestion
        const removedShip = this._ships.findIndex(el => {
```

Reply:
Ah yeah, I'm not sure what's best here when it comes to being consistent (always using the ()) or being minimalist and leaving it off, I take it it's best to leave off?

Reply:
Its only needed if there are multiple parameters, so when its a single they can always be removed